package.json

```
@@ -25,8 +25,7 @@
     "clean": "rm -rf packages/*/tsconfig.tsbuildinfo && rm -rf coverage test-report && lerna clean --yes && rm -rf node_modules && yarn cache clean",
     "lint:ts": "tsc --noEmit",
     "format": "prettier .",
-    "yarn": "yarn",
-    "build": "yarn tsc --build",
+    "build": "lerna run build",
```

This was working as intended using tsconfig paths/references
```suggestion
    "build": "yarn tsc --build",
```

---

It was definitely NOT working for me. Nothing was building at all, that's why I changed it. I wouldn't have noticed if it was working.

---

What were you seeing then? It is using the references to build packages in order based on any dependencies.